### ISSUE TITLE
Deal with lowercase vs. uppercase issue for sequence IDs

### ISSUE THREAD

**Issue by liammulh** [numberscope/backscope#52]
```
   id    |                                                         name                                                         
---------+----------------------------------------------------------------------------------------------------------------------
 a000040 | The prime numbers.
 A262827 | Number of ordered ways to write n as w^2 + x^3 + y^4 + 2*z^4, where w, x, y and z are nonnegative integers.
 a109326 | Smallest positive number that requires n steps to be represented as a sum of palindromes using the greedy algorithm.
 a000041 | a(n) is the number of partitions of n (the partition numbers).
(4 rows)
```

It seems like OEIS sequence IDs are being put into the database according to the user's capitalization.

**Comment by liammulh** [comment]
Uh oh, the OEIS IDs are being inserted with a leading space:

```
   id    
---------
 a000040
 A262827
 a109326
 a000041
```

Not sure how that's happening, but it will need to be fixed.

**Comment by liammulh** [comment]
I am confused:

```
scope=# SELECT id FROM sequences;
   id    
---------
 a000040
 A262827
 a109326
 a000041
(4 rows)

scope=# DELETE FROM sequences WHERE id = a000040;
ERROR:  column "a000040" does not exist
LINE 1: DELETE FROM sequences WHERE id = a000040;
                                         ^
scope=# DELETE FROM sequences WHERE id = "a000040";
ERROR:  column "a000040" does not exist
LINE 1: DELETE FROM sequences WHERE id = "a000040";
```

**Comment by liammulh** [comment]
It was a syntax issue:

```
scope=# DELETE FROM sequences WHERE id='a109326';
DELETE 1
```

**Comment by gwhitney** [comment]
Resolved by #56.
